assets/sections.js

```diff
@@ -2,203 +2,127 @@ angular.module('cafapi.templates', []).run(['$templateCache', function($template
   'use strict';
```

Comment:
Why are there any changes to sections.js?
Should the changes not only be within auditing.md